src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -2627,7 +2627,8 @@ def _ensure_default_log_analytics_workspace_for_monitoring(cmd, subscription_id,
         workspace_region = AzureFairfaxRegionToOmsRegionMap.get(rg_location, "usgovvirginia")
         workspace_region_code = AzureFairfaxLocationToOmsRegionCodeMap.get(workspace_region, "USGV")
     else:
-        logger.error("AKS Monitoring addon not supported in cloud : %s", cloud_name)
+        workspace_region = rg_location
+        workspace_region_code = rg_location.upper()
```

Comment:
Could you please help explain more why this change is to support USNat and USSec ?

Reply:
Right now we have a check for cloud that maps region to short code for workspace creation. Since we cannot add the mapping in the open source repo, i am using the approach of using the region name in the workspace which enables the support for the USNat and USSec clouds.